pages/dkp/kommander/2.1/install/small-environment/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: layout.pug
+navigationTitle: Install on small environment
+title: Install Kommander on a small environment
+menuWeight: 40
+excerpt: Install Kommander on a small environment
+beta: false
+enterprise: false
+---
+
+You can install Kommander on a small environment with smaller memory, storage, and CPU requirements for testing and demo purposes. This topic describes methods for installing Kommander in these environments. Refer to [Kommander documentation][2.1] for more information.
+
+You can install this minimal setup on networked and air-gapped environments, regardless of the license type (Essential or Enterprise).
+
+## Prerequisites
+
+Ensure you have done the following:
+
```

Comment:
the 2.2 page has this additional item which I believe would apply here, too:
```suggestion

- You have created a Kubernetes cluster.
```

Reply:
Craig mentioned that by stating we require a Konvoy cluster, it already covers it being a K8s cluster, so we were thinking about deleting this sentence on both topics.